config/deploy/staging/secrets.ejson

```diff
@@ -24,6 +24,12 @@
         "sendgrid_webhook_username": "EJ[1:KAN7c51q2VWDrnvvVz2B0EuxBXk67FXswztYOJHRdm0=:elop/93FDinCvVXMwDT9eRNVkQkNe3Cg:VHnaUh3gqhrkbTW76SetCcDQjCFTxtviu+bbYcMFoTPb60QQH3131dmgV2zSWA==]",
         "sendgrid_webhook_password": "EJ[1:KAN7c51q2VWDrnvvVz2B0EuxBXk67FXswztYOJHRdm0=:Jc7/IHv1SPsVn1pje8OZFpyI0+sRI9FQ:79/96QQJ3kfET2h2eM2eSUh0ZI1e/odfGzo61HaafHIgNeanDK3HmlDClJczhFr4kfjBimjeaab9Cp8=]"
       }
+    },
+    "nginx-basic-auth": {
+      "_type": "Opaque",
+      "data": {
+        ".htpasswd": "EJ[1:MVylFycMlDK6oU5Jje1iXf6AMMiHkYAD5RnXlXfg5Fs=:YcD96nyUoPoeK9fJgh5Uj9oaKtx69HqV:tL7FPskgJqmU5hTYrL5kPwnTzC0gITPxGpswSrS4AihF5/96GbPqS97Fz/qNr6YUsO3d4IcfClvbHapUzc7mYeW4dTlf3Q==]"
```

Comment:
> "EJ[1:MVylFycMlDK6oU5Jje1iXf6AMMiHkYAD5RnXlXfg5Fs=:YcD96nyUoPoeK9fJgh5Uj9oaKtx69HqV:tL7.."

Note this has two `:` in the value as htpasswd file is of the format `user:pass_hash`. I am hoping ejson won't get confused.